### ISSUE TITLE
create list of photos from old birdwalker database

### ISSUE THREAD

**Issue by wfwalker** [wfwalker/ebird-mybird#14]
the query is:

select locations.name, taxons.abbreviation, photos.taxon_latin_name, photos.original_filename, trips.date from photos, locations, taxons, trips where trips.id=photos.trip_id AND photos.location_id=locations.id AND taxons.latin_name=photos.taxon_latin_name;

**Comment by wfwalker** [comment]
and then mimic this code:

  def image_filename
    result = self.trip.date.to_s

```
raise ("missing taxon %s for photo %d" % [self.taxon_latin_name, self.id]) unless self.taxon

if self.taxon.abbreviation
  result = result + "-" + self.taxon.abbreviation
else
  result = result + "-" + self.taxon.latin_name.downcase.gsub(' ', '_')
end

if self.original_filename != nil
  result = result + "-" + self.original_filename
end

result = result + ".jpg"
```

  end

## 

So it's YYYY-MM-DD then abbreviation or latin-name then original filename

**Comment by wfwalker** [comment]
fixed by 2efe7082d3206ba2ac4eb4a04203282825419f87
